src/modules/access-token/internal/access-token-writer.ts

```diff
@@ -0,0 +1,27 @@
+import jsonwebtoken from 'jsonwebtoken';
+import ConfigService from '../../config/config-service';
+import { AccessToken, CreateAccessTokenParams } from '../types';
+
+export default class AccessTokenWriter {
+  public static createAccessToken(
+    params: CreateAccessTokenParams,
+  ): AccessToken {
+    const jwtToken = jsonwebtoken.sign(
+      { accountId: params.accountId },
+      ConfigService.getStringValue('jwt.token'),
+      { expiresIn: ConfigService.getStringValue('jwt.expiresIn') },
+    );
+    const accessToken = new AccessToken();
+    accessToken.accountId = params.accountId;
+    accessToken.token = jwtToken;
+
+    const vetifiedToken: jsonwebtoken.JwtPayload = jsonwebtoken.verify(
+      jwtToken,
+      ConfigService.getStringValue('jwt.token'),
```

Comment:
This is being read twice in this method. Consider reading it once and store in var.